src/components/custom/_dhero.js

```diff
@@ -128,17 +128,18 @@ const GoToLiveDemo = styled(LinkButton)`
     color: var(--color-white);
     border-color: var(--color-black-5);
     margin-left: 1.6rem;
-    padding: 1.1rem 1.6rem;
+    padding: 1.2rem 1.6rem;
     width: auto;
 
     @media ${device.laptopM} {
         font-size: 14px;
-        margin-left: 0;
     }
     @media ${device.mobileL} {
         max-width: 100%;
         padding: 10px 11px;
+        height: 42px;
```

Comment:
can we use padding only for both buttons, instead of adding fix height?

Reply:
I saw there was a fixed height for the other button in case of Mobile. If I change it to padding I might need to change it in the other button as well. Will try to look into that ✌🏻🙏🏻